docs/data/material/getting-started/templates/dashboard/components/AppNavbar.js

```diff
@@ -68,7 +68,7 @@ export default function AppNavbar() {
               Dashboard
             </Typography>
           </Stack>
-          <ColorModeIconDropdown />
+          <ColorModeIconDropdown data-screenshot="" />
```

Comment:
What is this for?

Reply:
Ok. I didn't found any other occurrence of `data-screenshot=""`, only `data-screenshot="toggle-mode"` and `data-screenshot="toggle-default-theme"`, that's why I was asking.

Reply:
to remove the `data-screenshot` from this button because in the page there are two of them (desktop and mobile). Playwright was not happy with it.